### DISCUSSION TITLE
Google Sign in / Auth

### DISCUSSION THREAD

**Discussion by ghost** [serverpod/serverpod#1945] [Q&A] [answered]
I am trying to use the google sign in for authentication. Everything works well , But after validating user account it shows like this
"serverpod_auth_google: ServerpodClientException: Internal server error. 
Call log id: 11, statusCode = 500" and explain how to retrieve the google user information ..

**Comment by Isakdl** [comment, marked as answer]
Could you post the stack trace from the server as well?

**Comment by ghost** [comment]
i am sorry , i forgot place google_client_secret.json in config folder. After placing that , it  resolves the issue . Thank You..
